flash_image/flash_mfi.rst

```diff
@@ -3,4 +3,70 @@
 How to flash MFI
 ################
 
-TODO
+1. Parepare mfi image on your host computer.
+--------------------------------------------
+
+i) Download mfi image.  
+
+ a.   Go to the `device web <https://www.adlinktech.com/Products/ROS2_Solution/ROS2_Controller/ROScube-X?lang=en/>`_.  
+ b.  Login or create an account.  
+ c. Click BSP to download the BSP MFI Binary  
+
+ii) Install required tools.
+
+.. code-block:: bash
+
+  $> sudo apt install python qemu-user-static
+
+iii) Check ROScube is in recovery mode now.
+
+.. code-block:: bash
+
+  $> lsusb | grep NVIDIA
```

Comment:
You can remove "$>" for users to copy easily.

Reply:
Remove done.